fuel-crypto/Cargo.toml

```diff
@@ -22,16 +22,10 @@ serde = { version = "1.0", default-features = false, features = ["derive"], opti
 sha2 = { version = "0.10", default-features = false }
 zeroize = { version = "1.5", features = ["derive"] }
 
-# Note that, while we don't use this dependency directly, we must specify it in
-# order to enable the transitive dependency's "js" feature which is required
-# for supporting the wasm32-unknown-unknown target.
-[target.'cfg(target_arch = "wasm32")'.dependencies]
```

Comment:
I'm only trying to figure out this part. @mitchmindtree Added it before=) He may know if we still need it or not. Usually, the `js` feature is required if you try to use the `std` code in the WASM.

`cargo check --target wasm32-unknown-unknown -p fuel-crypto --no-default-features` compiles successfully for me

Reply:
I think this was potentially to help address the wasm issues the indexer was having? Does it still compile to wasm with `--features random` enabled?

Reply:
> Does it still compile to wasm with --features random enabled?

Yeah, it compiles with `cargo build --target wasm32-unknown-unknown -p fuel-crypto --no-default-features --features random`